src/components/avatar/AvatarInitials.js

```diff
@@ -0,0 +1,59 @@
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import Box, { pickBoxProps } from '../box';
+import cx from 'classnames';
+import theme from './theme.css';
+import { Heading4 } from '../..';
```

Comment:
Not sure about this path syntax, so this is exported from the index file from 2 folders higher?

Reply:
I'd import that component from its own file, importing everything from one index file (that probably imports and exports this component too) can give circular dependencies, which can be quite annoying if you're more strict environments